source/_integrations/notify.mqtt.markdown

```diff
@@ -0,0 +1,202 @@
+---
+title: "MQTT notify"
+description: "Instructions on how to integrate MQTT notify entities into Home Assistant."
+ha_category:
+  - Notifications
+ha_release: 2024.5.0
+ha_iot_class: Configurable
+ha_domain: mqtt
+---
+
+The `mqtt` notify platform lets you send an MQTT message when the `send_message` service is called. This can be used to expose some service of a remote device, that allows to process a message, e.g. to show it on a screen.
```

Comment:
```suggestion
The **MQTT notify** platform lets you send an MQTT message when the `send_message` service is called. This can be used to expose a service of a remote device that allows processing a message, such as showing it on a screen.
```